src/components/page-configuration/overview.tsx

```diff
@@ -0,0 +1,112 @@
+import { List, Tooltip, Typography } from '@equinor/eds-core-react';
+import { AuthCodeMSALBrowserAuthenticationProvider } from '@microsoft/microsoft-graph-client/authProviders/authCodeMsalBrowser';
+import * as PropTypes from 'prop-types';
+import { useEffect, useRef, useState } from 'react';
+import { AsyncState } from '../../effects/effect-types';
+import { RequestState } from '../../state/state-utils/request-states';
+
+import Alert from '../alert';
+import SimpleAsyncResource from '../async-resource/simple-async-resource';
+import { useAuthentication } from '../graph/authentication';
+import { getGroup } from '../graph/graphService';
+
+interface OverviewProps {
+  adGroups?: string[];
+  appName: string;
+}
```

Comment:
export main prop interface
```suggestion
export interface OverviewProps {
  adGroups?: Array<string>;
  appName: string;
}
```